docs/upcoming_changes/9417.new_feature.rst

```diff
@@ -0,0 +1,4 @@
+Support math.log2
+=================
+
+Support for ``np.log2`` is added to CUDA target.
```

Comment:
This adds more than just `np.log2` support?

Reply:
Fixed